.github/workflows/CI.yaml

```diff
@@ -0,0 +1,39 @@
+name: Event Connect - CI
+on: [pull_request]
+jobs:
+  event-connect-build:
+    env:
+      MAVEN_OPTS: "-Dhttps.protocols=TLSv1.2 -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.showDateTime=true -Djava.awt.headless=true"
+    concurrency:
+      group: event-connect-pr-${{ github.head_ref }}
+      cancel-in-progress: true
+    timeout-minutes: 45
+    runs-on: ubuntu-latest
+    name: Build and Test
+    steps:
+      - name: Set up GraalVM
+        uses: DeLaGuardo/setup-graalvm@4.0
+        with:
+          graalvm: '21.1.0'
+          java: 'java11'
```

Comment:
Please replace this step with [these other steps](https://github.com/kiegroup/kogito-runtimes/blob/main/.github/workflows/apps-pr.yml#L31-L51) to:
- free a lot of space on the executor
- setup java 11 with the "blessed" `setup-java` action